lxd/storage/drivers/interface.go

```diff
@@ -28,7 +28,12 @@ type Driver interface {
 	// Pool.
 	Create() error
 	Delete(op *operations.Operation) error
+	// Mount mounts a storage pool if needed, returns true if we caused a new mount, false if
+	// already mounted.
```

Comment:
Can probably do without the wrapping, might shorten by just saying `false if not`.